Linux/Car_Client/chronos.cpp

```diff
@@ -13,7 +13,6 @@ Chronos::Chronos(QObject *parent) : QObject(parent)
 	mObjectState = ISO_OBJECT_STATE_OFF;
 
     mHeabPollCnt = 0;
-    mChronos->setTransmitterId(9); //TODO: Set this properly
```

Comment:
The RC car isn't lacking TxID at startup without this?

Reply:
Upon receiving an osem message this happens: 
mChronos->setTransmitterId(osem.transmitterID);

So I think it is redundant to set transmitterid to hardocded 9, since it will be overwritten when osem arrives.